config/storage.yml

```diff
@@ -15,11 +15,11 @@ local:
 #   bucket: your_own_bucket
 
 # Remember not to checkin your GCS keyfile to a repository
-# google:
-#   service: GCS
-#   project: your_project
-#   credentials: <%= Rails.root.join("path/to/gcs.keyfile") %>
-#   bucket: your_own_bucket
+google:
+   service: GCS
+   project: "My First Project"
+   credentials: <%= Rails.root.join("keyfiles/gcskeyfile.json") %>
```

Comment:
this is fine for now, but when you push to heroku this'll break since you won't have the keyfile up there. Looks like you can provide a hash here instead: http://edgeguides.rubyonrails.org/active_storage_overview.html#google-cloud-storage-service. With that, each config item (eg: `project_id`, `private_key_id`) can be saved as a Heroku config variable, and this code here can run checks like `project_id: ENV["gcs_project_id"] || Rails.application.secrets[:gcs_project_id]`.

That implies a bit more config setup making use of the `secrets.yml` file to store secretes locally, but should all be pretty easy to hook up.